microsoft/testsuites/security/openssl.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+
+from assertpy import assert_that
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.tools.openssl import OpenSSL
+
+
+
+class OpenSSLTestSuite(TestSuite):
+    """
+    Test suite for OpenSSL functionality.
+    """
+    def openssl_test_encrypt_decrypt(log: Logger, node: Node) -> None:
```

Comment:
It's a help method, and used in this class only. Change it to a private method, and move to below the public test methods.